dev/com.ibm.ws.jsp.2.3_fat/fat/src/com/ibm/ws/jsp23/fat/tests/JSPTests.java

```diff
@@ -336,9 +330,8 @@ private void nonReservedWordsHelperMethod(WebConversation wc, String path, Strin
      *                       if something goes horribly wrong
      */
     @Test
-    @SkipForRepeat("CDI-2.0") // No need to run against cdi-2.0 since these tests don't use CDI at all.
     public void testServlet() throws Exception {
```

Comment:
Why were these SkipForRepeat annotations removed in this class?

Reply:
All of the tests within this class do not use CDI, so I removed SkipForRepeat("CDI-2.0") from each method and added it to the class itself.